Content.Shared/Radiation/Components/GeigerComponent.cs

```diff
@@ -83,6 +79,31 @@ public sealed partial class GeigerComponent : Component
     ///     Played only for current user.
     /// </summary>
     public EntityUid? Stream;
+
+    /// <summary>
+    ///     Controls whether the geiger counter plays only for the local player, or plays for everyone nearby.
+    ///     Useful for things like hardsuits with integrated geigers. Alternatively, to create stationary radiation alarm objects.
+    /// </summary>
+    [DataField]
+    public bool LocalSoundOnly = false;
+
+    /// <summary>
+    ///     Used to control how loudly the geiger counter ticks.
+    /// </summary>
+    [DataField]
+    public float VolumeParameters = -4f;
+
+    /// <summary>
+    ///     Controls how quickly the sound gets quieter as a player moves away from the geiger counter.
+    /// </summary>
+    [DataField]
+    public float RolloffFactorParameters = 4f;
+
+    /// <summary>
+    ///     Controls the maximum distance that players can hear geiger counter ticks.
+    /// </summary>
+    [DataField]
+    public float MaxDistanceParameters = 10f;
```

Comment:
AudioParams are serializable, use that instead.
```suggestion
    public float AudioVolume = -4f;

    /// <summary>
    ///     Controls how quickly the sound gets quieter as a player moves away from the geiger counter.
    /// </summary>
    [DataField]
    public float AudioRolloffFactor = 4f;

    /// <summary>
    ///     Controls the maximum distance that players can hear geiger counter ticks.
    /// </summary>
    [DataField]
    public float AudioMaxDistance = 10f;
```